_maps/RandomZLevels/moonoutpost19.dmm

```diff
@@ -1970,33 +1970,21 @@
 	},
 /area/awaymission/moonoutpost19/syndicate)
 "dE" = (
-/obj/effect/decal/cleanable/blood/tracks{
-	desc = "Your instincts say you shouldn't be following these.";
-	dir = 8;
-	icon_state = "ltrails_1"
-	},
+/obj/effect/decal/cleanable/blood/footprints/tracks/wheels,
```

Comment:
also I'm pretty sure most of these got fucked by my 'just make it fucking compile' but I don't think people will really care that much tbh